Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM starefossen/github-pages:onbuild
+# Build the docs
+FROM starefossen/github-pages:147 AS build_docs
+COPY . /source
+RUN jekyll build --source /source --destination /site
 
-CMD bundle update && bundle exec jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
+# Reset with nginx
+FROM nginx:alpine
+COPY nginx.conf /etc/nginx/nginx.conf
+COPY --from=build_docs /site /usr/src/app/target
+CMD echo "Docker docs are viewable at:" && echo "http://0.0.0.0:4000" && nginx -g 'pid /tmp/nginx.pid;'
```

Comment:
Probably needs `exec "nginx -g 'pid /tmp/nginx.pid;'"` at the end, so that `nginx` becomes the main process.

Do you know where the `/tmp/nginx.pid` came from? The default `CMD` of the `nginx:alpine` image is:

```Dockerfile
CMD ["nginx", "-g", "daemon off;"]
```

So, this should do the same:

```Dockerfile
CMD echo -e "Docker docs are viewable at:\nhttp://0.0.0.0:4000"; exec nginx -g 'daemon off;'
```